src/App.js

```diff
@@ -19,6 +20,7 @@ function App() {
         <Route path="/articles" element={<Articles />}/>
         <Route path="/comments" element={<Comments />}/>
         <Route path="/topics" element={<Topics />}/>
+        <Route path="/topics/:topic" element={<TopicArticles />}/>
```

Comment:
I would suggest calling this the same name as the item that comes back from the backend, topic_slug. This way you've got less chance of there being a discrepancy somewhere.